controllers/transfer.rb

```diff
@@ -0,0 +1,73 @@
+class CustomWizard::TransferController < ::ApplicationController
+  before_action :ensure_logged_in
+  before_action :ensure_admin
+
+  skip_before_action :check_xhr, :only => [:export]
+
+
+  def index
+
+  end
+
+  def export
+
+    wizards = params['wizards']
+    wizard_objects = []
+    wizards.each do
+    |w|
```

Comment:
Minor formatting note, piped parameters should be on the same line as the method declaration, i.e.

```
wizards.each do |w|

end
```